pkgs/top-level/all-packages.nix

```diff
@@ -20798,6 +20798,10 @@ with pkgs;
 
   ghcid = haskellPackages.ghcid.bin;
 
+  gr-framework = libsForQt5.callPackage ../development/libraries/gr-framework {
+    stdenv = if stdenv.isDarwin then darwin.apple_sdk_11_0.stdenv else stdenv;
```

Comment:
```suggestion
    inherit (darwin.apple_sdk_11_0) stdenv;
```

Reply:
As I said, this doesn't work on Linux:

```
~> nixpkgs (gr-framework+1) nix build -Lv .#gr-framework
warning: Git tree '/home/paveloom/Playground/nixpkgs' is dirty
error:
       … while calling the 'derivationStrict' builtin

         at //builtin/derivation.nix:9:12: (source not available)

       … while evaluating derivation 'gr-framework-0.72.7'
         whose name attribute is located at /nix/store/i0lk1gmb1phln3i4a0zfps401rn69iqp-source/pkgs/stdenv/generic/make-derivation.nix:303:7

       … while evaluating attribute 'stdenv' of derivation 'gr-framework-0.72.7'

         at /nix/store/i0lk1gmb1phln3i4a0zfps401rn69iqp-source/pkgs/stdenv/generic/make-derivation.nix:332:14:

          331|       args = attrs.args or ["-e" (attrs.builder or ./default-builder.sh)];
          332|       inherit stdenv;
             |              ^
          333|

       (stack trace truncated; use '--show-trace' to show the full trace)

       error: Package ‘rewrite-tbd-20201114’ in /nix/store/i0lk1gmb1phln3i4a0zfps401rn69iqp-source/pkgs/os-specific/darwin/rewrite-tbd/default.nix:27 is not available on the requested hostPlatform:
         hostPlatform.config = "x86_64-unknown-linux-gnu"
         package.meta.platforms = [
           "x86_64-darwin"
           "i686-darwin"
           "aarch64-darwin"
           "armv7a-darwin"
         ]
         package.meta.badPlatforms = [ ]
       , refusing to evaluate.

       a) To temporarily allow packages that are unsupported for this system, you can use an environment variable
          for a single invocation of the nix tools.

            $ export NIXPKGS_ALLOW_UNSUPPORTED_SYSTEM=1

        Note: For `nix shell`, `nix build`, `nix develop` or any other Nix 2.4+
        (Flake) command, `--impure` must be passed in order to read this
        environment variable.

       b) For `nixos-rebuild` you can set
         { nixpkgs.config.allowUnsupportedSystem = true; }
       in configuration.nix to override this.

       c) For `nix-env`, `nix-build`, `nix-shell` or any other Nix command you can add
         { allowUnsupportedSystem = true; }
       to ~/.config/nixpkgs/config.nix.
```

Reply:
That's a regression, that shouldn't happen.